test-app/app_stm32wb.c

```diff
@@ -39,11 +39,17 @@ void main(void) {
     uint32_t updv;
     hal_init();
     boot_led_on();
+    uart_init(115200, 8, 'N', 1);
```

Comment:
```
app_stm32wb.c:42:5: warning: implicit declaration of function 'uart_init'; did you mean 'hal_init'? [-Wimplicit-function-declaration]
   42 |     uart_init(115200, 8, 'N', 1);
      |     ^~~~~~~~~
      |     hal_init
app_stm32wb.c:48:5: warning: implicit declaration of function 'uart_tx' [-Wimplicit-function-declaration]
   48 |     uart_tx('*');
      |     ^~~~~~~
```

Reply:
I've added a new header "uart_drv.h" to keep the declarations for the uart driver used in the test app here (which is an optional extension of HAL).